src/components/preview-flag-round/PreviewFlagRound.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from "react";
+import Flag from "../flag";
+import { IFlag } from "../flag/IFlag";
+
+const PreviewFlagRound: FC<IFlag> = ({ countryCode }) => {
```

Comment:
This component is repeated and not actually needed, let's discuss on why the original is not being used.